src/constants/MarkdownFiles/posts/2025-06-09-dmp-25-AmanNaik-week01

```diff
@@ -0,0 +1,85 @@
+---
+title: "DMP ’25 Week 01 Update by Aman Naik"
+excerpt: "This week's focus was exploring the write-activity codebase, finding appropriate grammar correction models & understanding Abiword documentations."
+category: "DEVELOPER NEWS"
+date: "2025-06-09"
+slug: "2025-06-09-dmp-25-AmanNaik-week01"
+author: "Aman Naik"
+description: "Member and DMP'25 Contributor at SugarLabs"
+tags: "dmp25,writeactivity,write,sugarlabs,week01,amannaik247"
+image: "assets/Images/c4gt_DMP.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 01 Progress Report by Aman Naik
+
+**Project:** [Add an AI-assistant to the Write Activity](https://github.com/sugarlabs/write-activity/issues/52)  
+**Mentors:** [Walter Bender](https://github.com/walterbender), [Ibiam Chihurumnaya](https://github.com/chimosky).
+**Reporting Period:** 2025-06-02 - 2025-06-08  
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Understand write activity's codebase.
+- **Goal 2:** Explore pertinent grammar correction models for real time grammar correction
+- **Goal 3:** Understand the [Abiword](https://github.com/AbiWord) word-processor documentation
+
+---
+
+## This Week’s Achievements
+
+1. **Understood the activity's codebase**  
+   - Gained a clear understanding of the overall structure and logic of the Write activity's codebase.
+   - This includes identifying key components, data flow, and how the Abiword processor is leveraged for this activity.
+2. **Explored pertinent grammar correction models**  
+   - Shortlisted grammar correction models suitable for real time grammar correction and can be used with an open source software.
+   - Created API endpoints using Hugging Face spaces for quick testing in sugar.
+
+3. **Understood the Abiword processor**  
+   - Abiword is an open source word processor, which is leveraged by the write activity.
+   - It has a parser called 'link-grammar' that was recently developed, which might be useful for grammar correction as well.
+
+---
+
+## Challenges & How I Overcame Them
+
+- **Challenge:** Difficulty in finding reliable grammar correction models on hugging face which can also be compatible open source software.
```

Comment:
Add double-spaces here to break to a new line.

Before:
![Screenshot from 2025-06-11 07-46-51](https://github.com/user-attachments/assets/3507118f-eb76-4378-a976-d86df67159b5)

After:
![Screenshot from 2025-06-11 07-48-36](https://github.com/user-attachments/assets/c05965dd-af94-4477-a2b7-98c89e035f6e)